app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/edits/NoteEditsUploader.kt

```diff
@@ -103,7 +99,8 @@ class NoteEditsUploader @Inject constructor(
         }
     }
 
-    private fun uploadCreateNote(pos: LatLon, text: String): Note = notesApi.create(pos, text)
+    private fun uploadCreateNote(pos: LatLon, text: String): Note =
+        notesApi.create(pos.toLatLon(), text)
```

Comment:
this looks wrong. LatLon is already a LatLon, why does it have a method called `toLatLon`?

In any case, `notesApi` should accept a `de.westnordost.streetcomplete.data.osmnotes.LatLon`.